docs/dev_intro.md

```diff
@@ -1,39 +1,27 @@
 ---
 id: dev_intro
-title: Intro
+title: Intro to Development
 ---
 
-Welcome to brief introduction to development with HydraDX.
+The purpose of the `Build` section of the HydraDX documentation is to share technical knowledge with the community and to empower individual contributions. HydraDX is a community-driven project which invests heavily in incentivizing community involvement, and we especially appreciate technical contributions!
 
-By reading this introductory tutorial, you should have good understanding of Hydra project, its purpose, design and implementation details as well.
+All technical contributions which are aligned with the goals of HydraDX are eligible for generous rewards which are paid out as Treasury tips in HDX. You can find more information about our reward scheme under the following links:
 
+* [New Deal reward scheme announcement](https://hydradx.substack.com/p/incentivized-testnet-reward-scheme)
+* [Example tasks which brings rewards](/new_deal)
+* [How to claim Treasury tips](/tip_request)
 
-By the end, you should be able to develop your own pallet on top of hydra or to extend Hydra.
 
-This tutorial assumes previous Substrate and Rust knowledge.
+## How to Start
 
-We encourage and support the following course to obtain and learn Substrate fundamentals.
+HydraDX is powered by Substrate which is a cutting-edge blockchain framework built on Rust. Knowledge of Rust is therefore an important prerequisite for chain development. If you want to learn Rust, a good place to start is the ["Rust Book"](https://doc.rust-lang.org/stable/book/).
 
-[Acala Runtime Developer Academy](https://www.industryconnect.org/substrate-runtime-developer-academy/)
+A further requirement is basic knowledge of the Substrate framework. If you want to get up to speed quickly, we highly recommend you to subscribe to the [Acala Runtime Developer Academy](https://www.industryconnect.org/substrate-runtime-developer-academy/).
 
-To learn Rust - [The Book](https://doc.rust-lang.org/stable/book/).
```

Comment:
Why not the rust reference ?

Reply:
who needs a book to learn Rust?

jking, it's on L17

Reply:
ah ok. i missed that,.